test/cucumber/HomePage.feature

```diff
@@ -12,3 +12,14 @@ Feature: Home Page
     Given I am at the main page
     When I access the Residue Generator area
     Then I should be at the residue generator index page
+
+  Scenario: Access User Index page
+    Given I am at the main page
+    When I access the User area
+    Then I should be at the User Index page
+    
+  Scenario: Access New Residue Generator page
+    Given I am at the main page
+    And I access the Residue Generator area
+    When I access the New Residue Generator page
+    Then I should be at the New Residue Generator page
```

Comment:
As páginas poderiam ser parametrizadas, tal como: I am at the "User Index" page. 
Possibilita maior reuso dos steps.

Reply:
Nesses casos, acho melhor não parametrizarmos, pois se houver alguma alteração nos nomes das páginas, o teste para de pegar